test/itemlist_formaction.cpp

```diff
@@ -10,27 +10,164 @@
 
 using namespace newsbeuter;
 
-TEST_CASE("process_op()", "[itemlist_formaction]") {
+
+TEST_CASE("process_op(OP_OPEN)", "[itemlist_formaction]") {
 	controller c;
 	newsbeuter::view v(&c);
-	TestHelpers::TempFile browserfile;
-	std::unordered_set<std::string> url_set;
-
-	std::vector<std::shared_ptr<rss_feed>> feedlist;
+	TestHelpers::TempFile pagerfile;
 
 	std::string test_url = "http://test_url";
 	std::string test_title = "Article Title";
 	std::string test_author = "Article Author";
 	std::string test_description = "Article Description";
 	time_t test_pubDate = 42;
-	std::string test_puDate_str = "Thu, 01 Jan 1970 01:00:42 +0100";
+	char test_pubDate_str[128];
+	strftime(test_pubDate_str, sizeof(test_pubDate_str), "%a, %d %b %Y %H:%M:%S %z", localtime(&test_pubDate));
 
 	std::string pager_prefix_title = "Title: ";
 	std::string pager_prefix_author = "Author: ";
 	std::string pager_prefix_date = "Date: ";
 	std::string pager_prefix_link = "Link: ";
 
 	std::string line;
+	configcontainer cfg;
+
+	cache rsscache(":memory:", &cfg);
+	cfg.set_configvalue("pager", "cat %f > " + pagerfile.getPath());
+
+	std::shared_ptr<rss_feed> feed(new rss_feed(&rsscache));
```

Comment:
Apparently you wrote the same code in the previous PR and I missed that during review; sorry for that. Good thing the code isn't set in stone :)